src/modules/search/searchSagas.js

```diff
@@ -9,15 +9,17 @@
 import { take, call, put } from 'redux-saga/effects';
 
 import * as actions from './search';
-import * as api from './searchApi';
+import * as searchApi from './searchApi';
+import * as imageApi from '../image/imageApi';
+import * as audioApi from '../audio/audioApi';
 
 export function* search(query, type) {
   try {
     let searchResult;
     if (type === 'concept') {
-      searchResult = yield call(api.searchConcepts, query);
+      searchResult = yield call(searchApi.searchConcepts, query);
```

Comment:
Søkes det fremdeles for forklaringer via search-api modulen?

Reply:
Søket på forklaringer skal ikke være endret. Måtte endre navnet på import av searchApi fra api til searchApi fordi imageApi og audioApi blir brukt her.

Reply:
ja, ser det ikkje er endra, men kallet går via search-api så denne kunne også være gjort som du har gjort med image-api og audio-api